torch/_inductor/external_callable.py

```diff
@@ -0,0 +1,13 @@
+import inspect
+from typing import Any, Callable, List
+
+external_matmul: List[Callable[..., Any]] = []
```

Comment:
perhpas this belongs in `config.py` so that you can pass it as an arg to `torch.compile(options={"external_matmul": [...]})` 

This way it gets hashed and will properly invalidate caches.

Reply:
Moved to config.py